ledger-service/http-json/src/main/scala/com/digitalasset/http/PartiesService.scala

```diff
@@ -74,18 +74,27 @@ class PartiesService(
 
   private def findUnknownParties(
       found: Set[domain.PartyDetails],
-      requested: Set[domain.Party]
-  ): Set[domain.Party] =
-    if (found.size == requested.size) Set.empty[domain.Party]
-    else requested -- found.map(_.identifier)
+      requested: OneAnd[Set, domain.Party]
+  ): Set[domain.Party] = {
+    val requestedIds: Set[domain.Party] = requested.tail + requested.head
```

Comment:
@SamirTalwar I actually tried it:
```
    import OneAnd._
    val requestedIds: Set[domain.Party] = requested.toSet
```
got:
```
ledger-service/http-json/src/main/scala/com/digitalasset/http/PartiesService.scala:80: error: value toSet is not a member of scalaz.OneAnd[Set,com.digitalasset.http.domain.Party]
    val requestedIds: Set[domain.Party] = requested.toSet
```
I think it is because there is no `Traversable` type class instance for `Set`. So `Traversable` can't be derived for `OneAnd[Set, X]`.